ConwaysGameOfLife.Tests/UniverseHelper.cs

```diff
@@ -1,22 +1,27 @@
+using System;
+using System.Collections.Generic;
+
 namespace ConwaysGameOfLife.Tests
 {
     public static class UniverseHelper
     {
-        public static bool UniverseContentsAreEqual(Universe obj1, Universe obj2)
+        public static bool UniversesAreEqual(Universe obj1, Universe obj2)
         {
             var isSizeSame = (obj1.GridWidth == obj2.GridWidth) && (obj1.GridLength == obj2.GridLength);
             if (!isSizeSame)
             {
                 return false;
             }
-            // test location using new method - for loop using index
+            
             for (var x = 0; x < obj1.GridLength; x++)
             {
                 for (var y = 0; y < obj1.GridLength; y++)
                 {
-                    if (obj1.GetCellAtLocation(x, y) != obj2.GetCellAtLocation(x, y))
+                    var cell1 = obj1.GetCellAtLocation(x, y);
+                    var cell2 = obj2.GetCellAtLocation(x, y);
+                    if (cell1 != cell2)

```

Comment:
Is this check needed? I'd imagine it will always be false unless you pass `UniversesAreEqual` the same `obj1` and `obj2`